bindings/rust/src/lib.rs

```diff
@@ -665,6 +667,31 @@ macro_rules! sig_variant_impl {
             }
         }
 
+        #[cfg(feature = "serde")]
+        impl Serialize for SecretKey {
+            fn serialize<S: Serializer>(
+                &self,
+                ser: S,
+            ) -> Result<S::Ok, S::Error> {
+                serde_bytes::ByteBuf::from(self.serialize()).serialize(ser)
```

Comment:
I fail to see the `serde_bytes`' value. I mean that one can achieve the goal in the same amount of code (if not less;-) without introducing an external dependency. The seralization would look as `ser.serialize_bytes(&self.serialize())` (here and elsewhere) and deserialization as `let bytes: &[u8] = Deserialize::deserialize(deser)?;`. `serde_bytes` appears useful in `derive` scenarios, but here it's arguably an overkill...

Reply:
Good point! Updated.